.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,29 @@
+# Do not edit this file! Make a pull request on changing
+# github/workflows/changelog.yaml in
+# https://github.com/itk-dev/devops_itkdev-docker if need be.
+
+### ### Changelog
+###
+### Checks that changelog has been updated
+
+name: Changelog
```

Comment:
I think you need to change 
.editorconfig
to match: https://github.com/itk-dev/economics/blob/develop/.editorconfig

According to the templates these files should be 2-space indented instead of 4. E.g.
https://github.com/itk-dev/devops_itkdev-docker/blob/develop/github/workflows/changelog.yaml

Reply:
Generally all the .yml / .yaml files should be 2-space indented except for the files in config/ .
see https://github.com/itk-dev/devops_itkdev-docker/blob/develop/github/workflows/yaml.yaml#L14